jobs/transforms/subtopic_validation.py

```diff
@@ -78,3 +78,11 @@ def _get_change_domain_class(self, input_model):
             return subtopic_page_domain.SubtopicPageChange
         else:
             return None
+
+
+@validation_decorators.RelationshipsOf( # type: ignore[no-untyped-call, misc, name-defined]
```

Comment:
Again, why this ignore?

Reply:
For the decorator `@validation_decorators.RelationshipsOf, mypy was throwing Untyped decorator makes function "story_summary_model_relationships" untyped [misc]` and `jobs/transforms/story_validation.py:87: error: Call to untyped function "RelationshipsOf" in typed context [no-untyped-call]`. Hence added both